lib/ui/wallet/components/wallet_view.dart

```diff
@@ -107,7 +107,7 @@ class _UserTokensList extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return SizedBox(
-      height: 150,
+      height: 151,
```

Comment:
this was causing me a weird issue with a missing pixel on a token....

Reply:
magic number. 

hmmm 🥺